distribution/lib/Standard/Database/0.0.0-dev/src/Data/Column.enso

```diff
@@ -156,6 +156,21 @@ type Column
             new_type_ref = type_mapping.infer_return_type infer_from_database_callback op_kind [self]+operands new_expr
             Column.Value new_name self.connection new_type_ref new_expr self.context
 
+    ## PRIVATE
+
+       Creates a ternary operation with given kind and operand.
+
+       Arguments:
+       - op_kind: The kind of ternary operator.
+       - operand1: The second operand to the binary operator.
+       - operand2: The third operand to the binary operator.
+       - new_name: The name of the resulting column.
+    make_ternary_op : Text -> Column | Any -> Column | Any -> (Text | Nothing) -> Column
+    make_ternary_op self op_kind operand1 operand2 new_name=Nothing =
+        effective_new_name = new_name.if_nothing <|
+            self.naming_helpers.function_name op_kind [self, operand1, operand2]
+        self.make_op op_kind [operand1, operand2] effective_new_name
```

Comment:
Do we really need this helper instead of just using `make_op` directly?

Reply:
Removed.